tests/integration/test_show.py

```diff
@@ -95,3 +95,11 @@ def test_wrong_input(c):
         c.sql(f'SHOW COLUMNS FROM "{c.schema_name}"."table"')
     with pytest.raises(AttributeError):
         c.sql('SHOW TABLES FROM "wrong"')
+
+
+def test_show_tables_no_schema(c):
+    df = pd.DataFrame({"id": [0, 1]})
+    c.create_table("test", df)
+
+    actual_df = c.sql("show tables").compute()
+    assert len(actual_df[actual_df["Table"] == "test"]) > 0
```

Comment:
I think it might be easier to make this mimic the other show tests if we use a separate context:

```suggestion
def test_show_tables_no_schema():
    c = Context()

    df = pd.DataFrame({"id": [0, 1]})
    c.create_table("test", df)

    actual_df = c.sql("show tables").compute()
    expected_df = pd.DataFrame({"Table": ["test"]})
    assert_frame_equal(actual_df, expected_df)
```

Reply:
We would also need to import `Context` for this to work